dipy/tracking/tests/test_utils.py

```diff
@@ -353,11 +354,27 @@ def test_near_roi():
     npt.assert_array_equal(near_roi(streamlines, np.eye(4), mask),
                            np.array([False, True, False]))
 
+    # test for handling of various forms of null streamlines
+    # including a streamline from previous test because near_roi / tol
+    # can't handle completely empty streamline collections
+    streamlinesNULL = [np.array([[0., 0., 0.9],
```

Comment:
PEP8:

```suggestion
    streamlines_null = [np.array([[0., 0., 0.9],
```

Reply:
Fixed [here]( https://github.com/dipy/dipy/pull/2515/commits/65664c88663003f1b6f25b2c5d22c5a83f6cdbd3) I think.

However, [it seems the pep8 styling is off](https://github.com/dipy/dipy/pull/2515#issuecomment-1038463545)?  In my spyder IDE it didn't seem to have a problem with this, so I'm not entirely sure what the specific fix is.

Reply:
From [PEP8](https://www.python.org/dev/peps/pep-0008/#naming-conventions): "mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility."

Reply:
FWIW, I agree with the automated pep8 that those lines are under-indented. One more space :-)

Reply:
I could have sworn I fixed some of this in an earlier push.  In any case [I believe it is fixed now](https://github.com/dipy/dipy/pull/2515/commits/5a59df375109406b4131f9c9d1b692a377b53eab).  Apologies for the false starts on this.  I suppose that even after all these years of working with git I'm still a bit unpracticed.